core/controllers/subtopic_viewer.py

```
@@ -89,6 +112,15 @@ def get(self, topic_name: str, subtopic_id: int) -> None:
             'recorded_voiceovers': {'voiceovers_mapping': {}},
             'written_translations': {'translations_mapping': {}},
         }
+
+        assert self.normalized_request is not None
+        skill_ids = self.normalized_request.get('skill_ids')
+        if skill_ids is not None:
+            try:
+                skill_fetchers.get_multi_skills(skill_ids)
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)